core-bundle/contao/templates/twig/backend/component/message/_message.html.twig

```diff
@@ -2,7 +2,7 @@
 
 {% block message_component %}
     {% set success = success is defined ? success : true %}
-    {% set show_close_button = show_close_button is defined ? show_close_button : not success %}
+    {% set autoclose = success and (constant('class', app.user) != 'Contao\BackendUser' or app.user.doNotHideMessages|default(false)) %}
```

Comment:
The variable is not being output anywhere.

Reply:
fixed in 14ba24cafae801e9ea5d91c44ff1b621a60bc164